src/Examples/Advanced/DataRecon/econ_recon.csv

```diff
@@ -1,251 +1,251 @@
-247,396.9823134195443,22035.866965128727,15267.056380640277,469.46477048524855,26922738.72181641,22035.866965128727,396.9823134195443,22035.866965128727,20339.865854156065,532.023553584857,26327926.42978862,22035.866965128727,487.45578269668573,16449.10843057496,-72688.82334645474,485.408059225173,100080.7578515525,406.2929782352016,9.869465058344975,12187.144436212988,32.898216861149926,2383.475811590312,1429.427522616964,487.45578269668573,16449.10843057496,-65893.08999898288,690.6147597741218,100526.10535897482,406.2929782352016,9.869465058344975,12187.144436212988,32.898216861149926,2383.475811590312,1429.427522616964,111783754.69783862,optimal,0.0,472500000.0
-248,403.1557407364955,22378.544254460405,14999.353377273841,466.04872668453504,26923728.3645751,22378.544254460405,403.1557407364955,22378.544254460405,19771.559845335294,525.2702896432125,26329678.29118279,22378.544254460405,487.2072165719622,16440.720611037515,-72614.90610353346,487.703988295426,99694.86317038503,406.08579909262664,9.864432366622509,12180.929900717694,32.881441222075026,2382.260416539336,1428.6986210991597,487.2072165719622,16440.720611037515,-66119.14255620826,683.9755963075314,100134.85506213133,406.08579909262664,9.864432366622509,12180.929900717694,32.881441222075026,2382.260416539336,1428.6986210991597,106795033.6369353,optimal,1.0,477500000.0
-249,374.3269853241676,20778.305156742917,14633.82610912394,461.37013122035074,26920303.18210468,20778.305156742917,374.3269853241676,20778.305156742917,20264.51252780449,531.1324184893483,26327365.167447988,20778.305156742917,484.06437569728405,16334.66600636829,-72667.92601743314,486.0572844539332,100327.73596833867,403.4662503572969,9.800799603820973,12102.354044118267,32.66933201273658,2366.8931043227653,1419.4824759534044,484.06437569728405,16334.66600636829,-65505.48262228446,701.9707636957352,99721.76879064579,403.4662503572969,9.800799603820973,12102.354044118267,32.66933201273658,2366.8931043227653,1419.4824759534044,116996120.6492724,optimal,0.0,472500000.0
+0,507.26442246113226,28157.454253341177,15314.526069598609,470.0696346476585,26921593.052197795,28157.454253341177,507.26442246113226,28157.454253341177,20132.867987560156,529.5744365345437,26247853.031979878,28157.454253341177,589.4439045285998,19890.706466031552,-72472.1047413791,492.1356618030969,98913.35808986446,491.3004497109795,11.934423879618931,14737.02442068278,39.781412932063105,2882.163366927972,1728.502391898142,589.4439045285998,19890.706466031552,-65651.21869775599,697.7051613288322,99731.31316956625,491.3004497109795,11.934423879618931,14737.02442068278,39.781412932063105,2882.163366927972,1728.502391898142,135672242.1319584,optimal,24.0,592500000.0
```

Comment:
Since the data is written to CSV and the changes are very minor, I assume this is fine.